features/steps/steps.py

```diff
@@ -1,56 +1 @@
-# -*- coding: utf-8; -*-
-#
-# This file is part of Superdesk.
-#
-# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
-#
-# For the full copyright and license information, please see the
-# AUTHORS and LICENSE files distributed with this source code, or
-# at https://www.sourcefabric.org/superdesk/license
-
-from superdesk.tests.steps import apply_placeholders, json_match, get_json_data
-from superdesk.tests import set_placeholder
-from behave import when, then
-import json
-from wooper.general import (
-    get_body
-)
-
-
-@when('we save API token')
-def step_save_token(context):
-    context.headers.append(('Authorization', context.news_api_tokens.get('_id')))
-    return
-
-
-@when('we set header "{name}" to value "{value}"')
-def step_set_header(context, name, value):
-    context.headers.append((name, value))
-
-
-@then('we get headers in response')
-def step_assert_response_header(context):
-    test_headers = json.loads(apply_placeholders(context, context.text))
-    response_headers = context.response.headers
-    headers_dict = {}
-
-    for h in response_headers:
-        headers_dict[h[0]] = h[1]
-
-    for t_h in test_headers:
-        json_match(t_h, headers_dict)
-
-
-@then('we store NEXT_PAGE from HATEOAS')
-def step_store_next_page_from_response(context):
-    data = get_json_data(context.response)
-    href = ((data.get('_links') or {}).get('next_page') or {}).get('href')
-    assert href, data
-    set_placeholder(context, 'NEXT_PAGE', href)
-
-
-@then('we get "{text}" in text response')
-def we_get_text_in_response(context, text):
-    with context.app.test_request_context(context.app.config['URL_PREFIX']):
-        assert(isinstance(get_body(context.response), str))
-        assert(text in get_body(context.response))
+from newsroom.tests.steps import * # noqa
```

Comment:
just wondering why having tests in newsroom ?

Reply:
To have an ability to reuse a core's [contest](https://github.com/superdesk/newsroom-app/blob/9dfa7f740f8265429196aaaef4c886b6bf770b10/server/tests/conftest.py#L1), [fixtures](https://github.com/superdesk/newsroom-app/blob/9dfa7f740f8265429196aaaef4c886b6bf770b10/server/tests/fixtures.py#L1) and [steps](https://github.com/superdesk/newsroom-app/blob/9dfa7f740f8265429196aaaef4c886b6bf770b10/server/features/steps/steps.py) from the newsroom-app: